daemon/container_operations.go

```diff
@@ -420,31 +421,57 @@ func (daemon *Daemon) updateContainerNetworkSettings(ctr *container.Container, e
 	}
 }
 
-func (daemon *Daemon) allocateNetwork(ctx context.Context, cfg *config.Config, ctr *container.Container) (retErr error) {
-	if daemon.netController == nil {
-		return nil
+// initializeNetworking prepares network configuration for a new container.
+// If it creates a new libnetwork.Sandbox it's returned as newSandbox, for
+// the caller to Delete() if the container setup fails later in the process.
+func (daemon *Daemon) initializeNetworking(
+	ctx context.Context,
+	cfg *config.Config,
+	ctr *container.Container,
+) (newSandbox *libnetwork.Sandbox, retErr error) {
```

Comment:
Slightly tempted to keep this unwrapped; yes, it's a long line, but most of it can be skimped over, knowing that it's just the func's signature.

Reply:
Done. Is the rule "no line-wrapping in function declarations"?

(To me, it's just slightly harder to read and see the return types. And diffs won't pick out future changes as clearly.)

Reply:
Most of the codebase is written that way, but I admit I'm not a big fan of that. To @robmry's point, it's harder to read IMHO.

Reply:
Tempted to say; if the list of arguments for a function is becoming so long that it needs wrapping, that's a very good indicator that the signature may be in need of an overhaul 😄

Reply:
Two params and a return value plus context and error, seems ok? So we need shorter names?